specs/waku/v2/waku-spam-protected-relay.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Waku Spam Protected Relay
```

Comment:
I'd call this something like Waku RLN Relay or to that effect, otherwise seems like the existing relay isn't spam protected at all. Plus this opens up door for other spam protection mechanisms.

Reply:
Thanks! your suggested title makes so much sense, will apply the change.